### ISSUE TITLE
Frankfurt theme

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#80]
In the Frankfurt theme (and probably also in related themes), for the formating of the sections and bullets in the upper part only the length of section title is taken into account, not the number of slides (=number of bullets). Thus if the last section has a short title, the bullets are likely to run out of the slide. (A sample is attached.)

Once one has figured out the reason for this happening, there is of course an easy workaround: name the last section
````
\section{short~~~}
````
instead of 
````
\section{short}
````
However this is certainly not very elegant.
````
\documentclass{beamer}

\mode<presentation>
{
  \usetheme{Frankfurt}
  \setbeamercovered{transparent}
}

\title
{Title}

\author
{Author}

\institute
{Institute}

\date{Date}

%\AtBeginSection[]
%{
%  \begin{frame}<beamer>{Outline}
%    \tableofcontents[currentsection,subsectionstyle=show/show/hide]
%  \end{frame}
%}

\begin{document}

\begin{frame}
  \titlepage
\end{frame}

\begin{frame}{Outline}
  \tableofcontents[hideallsubsections]
\end{frame}

\section{A}

\subsection{A1}

\begin{frame}
Frame
\end{frame}

\section{B}

\subsection{B1}

\begin{frame}
Frame
\end{frame}

\begin{frame}
Frame
\end{frame}

\begin{frame}
Frame
\end{frame}

\begin{frame}
Frame
\end{frame}

\end{document}
````

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/80
- Originally reported by: switzel
- Originally created at: 2010-09-28

**Comment by josephwright** [comment]
_From Vedran Miletić on 2010-10-26_

Indeed. I will look into it eventually, but not for next release.

**Comment by josephwright** [comment]
Fixed by acdc90b51023eb7d5fa79836d8f980989a0bbefb
